wowchemy/test/config.yaml

```diff
@@ -1,11 +0,0 @@
-title: Wowchemy Test
```

Comment:
This file shouldn't be deleted...

Reply:
I replaced it with config.toml

some of the issues we faced were configuration related

they are now both included